src/components/KMultiselect/KMultiselect.vue

```diff
@@ -29,11 +29,12 @@
             return
           }"
           :position-fixed="positionFixed"
-          target=".multiselect-trigger"
+          :target="`[id='${multiselectWrapperId}']`"
```

Comment:
I think this can be easier
```suggestion
          :target="`#${multiselectWrapperId}`"
```